README.md

```diff
@@ -1,4 +1,8 @@
 # Cloud SQL Connector for Python Drivers
+![CI](https://storage.googleapis.com/cloud-devrel-public/cloud-sql-connectors/python/python3.10_linux.svg)
```

Comment:
I assume you'll have to update this one
```suggestion
![CI](https://storage.googleapis.com/cloud-devrel-public/cloud-sql-connectors/python/tests.svg)
```

Reply:
No need to update it actually. I only changed the name that appears on the badge, not the actual path to the badge :)